gravitee-apim-console-webui/src/management/api/api-traffic-v4/runtime-logs-settings/runtime-logs-message-settings/api-runtime-logs-message-settings.component.ts

```diff
@@ -221,6 +222,20 @@ export class ApiRuntimeLogsMessageSettingsComponent implements OnInit, OnDestroy
       });
   }
 
+  private handleTracingEnabledChanges(): void {
+    this.form
+      .get('tracingEnabled')
+      .valueChanges.pipe(takeUntil(this.unsubscribe$))
+      .subscribe((tracingEnabled) => {
+        if (!tracingEnabled) {
+          this.disableAndUncheck('tracingVerbose');
+        } else {
+          this.form.get('tracingVerbose').enable();
+          this.form.get('tracingVerbose').setValue(false);
```

Comment:
How about leaving it as is and just disabling the button? 
The idea is just to make the user understand that this button is not deactivated now. But not to change its configuration/value.
it's classic behavior. already present on other settings screen

Reply:
Good point. Let me update